framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/external/MavenRepositoryExternalDeployment.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.cloud.openshift.deployment.external;
+
+import org.kie.cloud.api.deployment.MavenRepositoryDeployment;
+
+public interface MavenRepositoryExternalDeployment<U> extends ExternalDeployment<MavenRepositoryDeployment, U> {
```

Comment:
Is this interface needed, when MAVEN_REPOSITORY is enum now?

Reply:
Right ... I removed it.